pkg/transportrequest/gitutils.go

```diff
@@ -97,3 +98,12 @@ func FindLabelsInCommits(commits object.CommitIter, label string) ([]string, err
 	sort.Strings(labels)
 	return labels, nil
 }
+
+func finishLabel(label string) string {
+	// contains prefix, like the old default
+	if strings.ContainsAny(label, ":=") {
```

Comment:
now let's hope the a user does not configure something without `=`, `:`. I guess basically nobody will re-configure our defaults. From that point of view we should use the approach here.